docs/workflows/create_sync_publish.rst

```diff
@@ -146,9 +146,13 @@ Sync repository ``foo`` using remote ``bar``
 .. literalinclude:: ../_scripts/sync.sh
    :language: bash
 
-You can specify ``mirror=True`` for a mirror mode. It means Pulp won't update
-repository using previous repository version but create a new copy of remote
-repository as a new repository version.
+There are 3 sync modes to chose from, using the ``sync_policy`` option.
+
+- ``additive`` (the default) will retain the existing contents of the repository and add the contents of the repository being synced.
+- ``mirror_content_only`` will create a repository which contains the exact same content as the one being synced - removing any existing content that isn't present in that repo.
+- ``mirror_complete`` will create an _exact_ bit-for-bit copy of the repository being synced, and will automatically create a publication. This will keep metadata checksums intact, but is not possible for all repositories, as some use features which are incompatible with local clones.
+
+The ``mirror`` option is deprecated, ``sync_policy`` should be used instead. If used, a value of ``True`` will change the default ``sync_policy`` to ``mirror_complete``.
```

Comment:
If ``mirror`` option  used, a value of ``True`` will change the default ``sync_policy`` to ``mirror_complete``. `Mirror`` set to ``False`` is equivalent to ``additive`` behavior.